azure-pipelines.yml

```diff
@@ -764,3 +764,88 @@ stages:
           displayName: 'Install dependencies'
         - bash: python -m unittest discover -v test/ipynb
           displayName: 'Run image test'
+
+schedules:
+- cron: "0 6 * * *"
+  displayName: Nightly slow build
+  branches:
+    include:
+      - master
+  jobs:
```

Comment:
Azure doesn't like this, it failed validation on this line. I'll have to read the docs some more about the syntax for scheduled jobs.